property.go

```diff
@@ -0,0 +1,91 @@
+package opslevel
+
+import "fmt"
+
+type PropertyDefinitionInput struct {
+	Name   string `json:"name"`
+	Schema string `json:"schema"`
```

Comment:
I think the type for `Schema` should be `JSON` with `json:"schema scalar:"true"`?
Similar to [InfrastructureResourceSchema](https://github.com/OpsLevel/opslevel-go/blob/main/infra.go#L10)

Reply:
I am not sure, I am looking at infra_test.go and I don't see any instances of the `JSON` type used in the tests. @rocktavious should we use the `JSON` type or is a string OK?

Reply:
So we need to ensure that whats gets sent to the API is "stringify" json so
```
{\"key\":\"value\"}
```

I'm pretty sure our JSON scaler type does that.  Additionally - they ended up making a new type in the API - `JSONString` IIRC - we may need to match that by making our own matching scalar type to ensure it works correctly.

For the end user using opslevel-go - i think it would be ideal for them to use `map[string]any`